manifests/init.pp

```diff
@@ -120,6 +136,11 @@
     amqp_truststore              => $::certs::candlepin::amqp_truststore,
     qpid_ssl_cert                => $::certs::qpid::client_cert,
     qpid_ssl_key                 => $::certs::qpid::client_key,
+    db_host                      => $katello::candlepin_db_host,
```

Comment:
Defaulting to undef would remove the need to copy defaults. That does mean changing the types to Optional. Additionally all these variables are already in the local scope so you can use ```$candlepin_db_host```.